### PR TITLE
Use PKCE flow

### PR DESCRIPTION
**Pull request by aarongranick-okta**
- Also adds instructions for running resource-server

### CODE REVIEW

File: custom-login/src/Login.jsx [hunk 1]
```diff
@@ -13,35 +13,46 @@
...
       },
       authParams: {
-        responseType: ['id_token', 'token'],
-        issuer: config.oidc.issuer,
+        grantType,
+        responseType,
+        issuer,
         display: 'page',
-        scopes: config.oidc.scope.split(' '),
+        scopes: scope.split(' '),
```
[the remark this answers is not in the document]
Comment (aarongranick-okta) [reply custom-login/src/Login.jsx]: removed completely (scope is deprecated), no need for this type of logic here anymore

File: custom-login/src/config.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ export default {
     issuer: ISSUER,
     redirectUri: 'http://localhost:8080/implicit/callback',
     scope: 'openid profile email',
+    pkce: true,
+    disableHttpsCheck: true,
```
[the remark this answers is not in the document]
Comment (aarongranick-okta) [reply custom-login/src/config.js]: removed

File: README.md [hunk 3]
```diff
@@ -26,10 +26,18 @@ Please find the sample that fits your use-case from the table below.
...
+```
+npm run resource-server
+```
+
 ## Running the tests
 
 In addition to the other environment vars, you will need credentials for a test user.
-Set the values for `USERNAME` and `PASSWORD`, either exported as shell variables, or as entries in your `testenv` file:
+Set the values for `USERNAME` and `PASSWORD` and export as shell variables. 
+(At this time, the `testenv` file is not supported)
```
Comment (swiftone) [inline README.md]: Do we have a Jira to change that?

File: okta-hosted-login/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Okta React + Okta Hosted Login Example
 
-This example shows you how to use the [Okta React Library][] to login a user to a React application.  The login is achieved through the [Implicit Flow][], where the user is redirected to the Okta-Hosted login page.  After the user authenticates they are redirected back to the application with an ID token and access token.
+This example shows you how to use the [Okta React Library][] to login a user to a React application.  The login is achieved through the [PKCE Flow][], where the user is redirected to the Okta-Hosted login page.  After the user authenticates they are redirected back to the application with an ID token and access token.
 
 This example is built with [Create React App][].
```
Comment (swiftone) [inline okta-hosted-login/README.md]: https://github.com/facebook/create-react-app

File: README.md (hunk 3 above)
Comment (aarongranick-okta) [reply README.md]: we do now! https://oktainc.atlassian.net/browse/OKTA-247107

**Pull request merged**